src/styles/Trips.css

```diff
@@ -28,4 +21,14 @@ h3 {
 
 a {
   text-decoration: none;
+  letter-spacing: 2px;
+}
+.randomImages__info {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 0;
+}
+
+.randomImages__delete {
```

Comment:
I think the button is pretty small riight now:
![image](https://user-images.githubusercontent.com/6980777/117700790-c57ee200-b1c6-11eb-8973-e198dbec33ed.png)